### PR TITLE
Create header

### PR DESCRIPTION
**Pull request by Philippevial**
Simple menu added with the logo.

### CODE REVIEW

**Comment by LordRekishi** [comment]
[Add header with menu and logo](https://trello.com/c/nFfUzfx2/75-add-header-with-menu-and-logo)

File: src/components/HamburgerMenu.vue [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<template>
+  <div class="menu-options">
```
Comment (LordRekishi) [inline src/components/HamburgerMenu.vue:2]: I suggest we don't use generic <div> tags, instead use <main> or <nav> or something like that instead.

File: src/components/HamburgerMenu.vue [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+<style scoped>
+a {
+  color: white;
+  text-decoration: none;
+  padding: 10px;
+}
+
+.menu-options {
+  display: flex;
+  flex-direction: row;
```
Comment (LordRekishi) [inline src/components/HamburgerMenu.vue:20]:
```suggestion
```
Row is the default flex direction, do we actually need to specify it here?

File: src/components/NavHeader.vue [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<template>
+  <div class="banner">
```
Comment (LordRekishi) [inline src/components/NavHeader.vue:2]: Same as before, best practice is to not use div if we have another choice. Main, section, article, nav, are some examples of better tags.

**Review by LordRekishi — CHANGES_REQUESTED** [review]
Some questions:

- Do we want to have the logo to the left of the nav bar links? Or above like now? Do we want to make it a bit smaller? 
- Do we need more space between the header and the landing page logo? 
- Do we want to separate the header with some kind of border or similar?

File: src/components/NavHeader.vue [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  height: 10vh;
+  width: 100%;
+  margin-bottom: 2rem;
+  position: relative;
+}
+
+img {
+  width: 12vh;
+  padding-top: 1rem;
+  padding-left: 1rem;
```
Comment (LordRekishi) [inline src/components/NavHeader.vue:25]: Here we set the width, but we use viewport-height as relative unit of measurement.

File: src/components/NavHeader.vue [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+<script setup>
+import HamburgerMenu from "./HamburgerMenu.vue";
+</script>
+
+<style scoped>
+.banner {
+  height: 10vh;
+  width: 100%;
+  margin-bottom: 2rem;
+  position: relative;
```
Comment (LordRekishi) [inline src/components/NavHeader.vue:19]: Do we want the banner to take up different amount of space depending on what kind of screen the user is using? Or do we want to set it to a specific height that stay constant.

Also, why do we use position relative here?

**Review by LordRekishi — CHANGES_REQUESTED** [review]
I suggest looking over the header in different view sizes and see what spacing and such we can change quickly to make it look a little bit better, other than that it all works as it should

File: src/components/NavHeader.vue (hunk 4 above)
Comment (LordRekishi) [reply src/components/NavHeader.vue:25]: Or is this viewport-horizontal?

**Comment by FelixJacobsen** [comment]
We see the changes you want but this code will be changed so much that we think it's just better to get it done and work from there.

**Pull request merged**